spec/controllers/api/v0/shipments_controller_spec.rb

```diff
@@ -246,6 +246,7 @@
 
       before do
         order.shipments.first.shipping_methods = [shipping_method1, shipping_method2]
+        order.shipments.each(&:refresh_rates)
```

Comment:
I think now, we don't need to use
```ruby
        order.shipments.first.shipping_methods = [shipping_method1, shipping_method2]
```
because this will create shipping rates with cost = 0.

Instead, we can use 
```ruby
        order.shipments.each(&:refresh_rates)
``` 
`refresh_rates` will automatically load the shipping methods from the distributor and calculate the cost before creating the `shipping_rates`.

https://github.com/openfoodfoundation/openfoodnetwork/blob/f122f4bf50e83e744e6527531a4688393498e4b3/app/models/spree/shipment.rb#L117-L121
https://github.com/openfoodfoundation/openfoodnetwork/blob/f122f4bf50e83e744e6527531a4688393498e4b3/engines/order_management/app/services/order_management/stock/estimator.rb#L18-L21